front-end/src/pages/main/meals/components/MealResults.jsx

```diff
@@ -20,13 +20,15 @@ function MealResults(props){
     const [apiData, fillData] = useState([]);
     const [searchQuery, setSearchQuery] = useState("");
     const [filterMap, setFilterMap] = useState({})
+    const [minCal, setMinCal] = useState(0);
+    const [maxCal, setMaxCal] = useState(3500)
```

Comment:
0 and 350 could be consts on their own in case the min or max ever changes. Also by giving it a variable name, the code is more readable.